app/src/main/java/com/gdfp/android_evaluationv2/MainActivity.java

```diff
@@ -89,10 +92,10 @@ public void onClick(View v) {
     public void onItemClick(View v, int position) {
         //TODO: Handle list item click event
         // Create an Intent to navigate to the TaskDetailActivity
-
+        Intent intent = new Intent(this,TaskDetailActivity.class);
```

Comment:
Just for keep our code little bit more readable please change this variable name to 
```detailIntent``` 
because is more understadable.